js/image/shared/ImageUploader.svelte

```diff
@@ -153,7 +154,8 @@
 	{/if}
 	<div class="upload-container">
 		<Upload
-			hidden={value !== null || active_tool === "webcam"}
+			hidden={value !== null ||
```

Comment:
Should we disable upload progress from the webcam in general? Still seeing it when you take a picture from webcam without streaming.

![webcam_upload_progress](https://github.com/gradio-app/gradio/assets/41651716/dc09b148-07b9-4ad5-b85d-aae9fca56c3e)

Other than that (and the type error) this looks good to me! Thanks for the fix

Reply:
I really think we should just use the local binary data for the preview image (same for all media) with a subtle uploading animation + block the submit button/ submit target until it is done.

For another PR though. Will open an issue.